src/vm/threadsuspend.cpp

```diff
@@ -8342,7 +8342,7 @@ void ThreadSuspend::SuspendEE(SUSPEND_REASON reason)
 #endif //TIME_SUSPEND
 }
 
-#if defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
+#if !defined(_TARGET_ARM64) && defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
I don't think we should exclude it for ARM64, we need to implement activation injection for it too.